lib/philomena_web/controllers/image/tag_change_controller.ex

```diff
@@ -35,4 +57,17 @@ defmodule PhilomenaWeb.Image.TagChangeController do
 
   defp added_filter(query, _params),
     do: query
+
+  defp log_details(conn, _action, %{image: image, details: details}) do
+    %{
+      body: "Deleted tag change (#{details}) from history",
+      subject_path: Routes.image_path(conn, :show, image)
+    }
+  end
+
+  defp tag_change_details(%TagChange{added: true, tag: tag}),
+    do: "added tag \"#{tag.name}\""
```

Comment:
maybe instead of `added/removed tag "tagname"` simply put it like `+tagname` or `-tagname`?

Reply:
I was wondering about this, I considered this but I wasn't sure if it was a good idea or not, glad to hear that it is indeed a good idea - done!